features/user_can_comment_on_an_article.feature

```diff
@@ -0,0 +1,37 @@
+Feature: User can comment on an article
+    As an Author,
+    In order to provide more value to my readers,
+    I would like to be able to attach an image to the article.
+
+    Background:
+        Background:
+        Given the date is "2018-08-18"
+        Given the following article exist
+        | title          | body                                                             | 
+        | This is so sad | A recent report suggest that news are mostly sad. Which is sad.  |
+        | This is so fun | A recent report suggest that news are mostly fun. Which is fun.  |
+
+        And the following user exists
+        | email             | role       |
+        | pablo@test.com    | basic_user |
+        | pablo2@test.com   | subscriber |
+
+        Scenario: Pablo can NOT see article page
+            When I am logged in as 'pablo@test.com'
+            And I am on the 'landing' page
+            And I click on "This is so sad"
+            Then I should see 'Access denied'
+            Then I should be redirected to the 'landing' page
+
+        Scenario: Pablo2 CAN see article page
+            When I am logged in as 'pablo2@test.com'
+            And I am on the 'landing' page
+            And I click on "This is so sad"
+            Then I should be on the show page for 'This is so sad'
```

Comment:
No need for assertions in the middle of the scenario. We know that this works....